website/templates/util/render_node.mako

```diff
@@ -58,7 +80,18 @@
                     <i class="fa fa-times remove-pointer" data-id="${summary['id']}" data-toggle="tooltip" title="Remove link"></i>
                     <i class="fa fa-code-fork" onclick="NodeActions.forkPointer('${summary['id']}', '${summary['primary_id']}');" data-toggle="tooltip" title="Fork this ${summary['node_type']} into ${node['node_type']} ${node['title']}"></i>
                 % endif
-                <i id="icon-${summary['id']}" class="pointer fa fa-angle-down" onclick="NodeActions.openCloseNode('${summary['id']}');" style="font-weight:bold;"></i>
+                <div class="dropdown pull-right">
+                  <button class="btn btn-default dropdown-toggle quickActions" type="button" data-toggle="dropdown">
+                    <span class="glyphicon glyphicon-option-horizontal"></span>
+                  </button>
+                  <ul class="dropdown-menu dropdown-menu-right" id="componentQuickActions">
+                    <li><a tabindex="-1" href="${domain}${summary['id']}/settings/">Settings</a></li>
+                    <li><a tabindex="-1" href="${domain}${summary['id']}/contributors/">Manage Contributors</a></li>
+                    % if 'admin' in user['permissions']:
```

Comment:
Was admin in spec or did it include write contrib? This could get tricky as you may need to assess if the person has write permissions on the component, not on the current project?